data/ui_scripts/custom_depot/scoreboard_override.lua

```diff
@@ -0,0 +1,35 @@
+local player_score = 0
+local player_old_score = 0
+
+Scoreboard_orig = LUI.MenuBuilder.m_types_build["scoreboard"]
+Scoreboard = function(unk1, unk2)
+    local scoreboard = Scoreboard_orig(unk1, unk2)
+
+    scoreboard:registerOmnvarHandler("ui_round_end", function(f22_arg0, f22_arg1)
+        if GameX.IsRankedMatch() then
+            local gamemode = GameX.GetGameMode()
+            local player_stats = Game.GetPlayerScoreInfoAtRank(Game.GetPlayerTeam(), Game.GetPlayerScoreRanking())
+
+            if Engine.GetDvarInt(string.format("scr_%s_halftime", gamemode)) == 1 or IsGameTypeRoundBased(gamemode) or
```

Comment:
scr_* dvars are only supposed to be used by the server, their value on the client won't match the value on the server

Reply:
^ to add on to this, the client has to know what these dvars are anyways (at least round/time limit) because the clientside lui scripting sets it to match the servers value.